test/csuite/schema_abort/main.c

```diff
@@ -909,11 +920,14 @@ main(int argc, char *argv[])
     verify_only = false;
     working_dir = "WT_TEST.schema-abort";
 
-    while ((ch = __wt_getopt(progname, argc, argv, "Ch:mT:t:vxz")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "Cch:mT:t:vxz")) != EOF)
         switch (ch) {
         case 'C':
             compat = true;
             break;
+        case 'c':
+            use_columns = true;
+            break;
```

Comment:
Since FLCS is also "columns", might be worth being more specific (not sure if this test can ever support FLCS)?

Reply:
It probably can't (not without a lot of surgery) but this raises another point: I added -c for columns to five tests, of which FLCS is plausible for 3-4 of them (assuming FLCS eventually supports timestamps and history, anyway)... so it might be better to use the prior art of -t c / -t r / -t f for this, except that for half of them -t already means something else. I could change the ones where it doesn't (wt6185 and truncated_log) to -t... or I could leave them all as is on the grounds that trying to maintain consistency of command-line arguments is a lost cause anyway. Suggestions?

And then on reflection since this boolean actually controls the use of recno keys, it would be set also for FLCS so I think it's better to leave its name as is, and to insert a comment here saying the test supports VLCS only. Am doing that.

Reply:
I agree with the plan, I suspect command-line consistency is a lost cause.